stage1/init/kvm/hypervisor/lkvm_driver.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build hv_lkvm
+
+package hypervisor
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+
+	"github.com/coreos/rkt/stage1/init/kvm"
+	"strings"
+)
+
+var (
+	driverConfiguration = KvmHypervisor{
+		Bin: "./lkvm",
+		KernelParams: []string{
+			"systemd.default_standard_error=journal+console",
+			"systemd.default_standard_output=journal+console",
+		},
+	}
+)
+
+func StartCmd(wdPath, name, kernelPath string, nds []kvm.NetDescriber, cpu, mem int64, debug bool) ([]string, error) {
```

Comment:
maybe group in separate lines for better readability (if that's what the newline has been for)